docs/Modern-NetworkLayer.md

```diff
@@ -66,7 +66,8 @@ import {
   RecordSource,
   Store,
 } from 'relay-runtime';
-import RelayQueryResponseCache from 'relay-runtime/lib/RelayQueryResponseCache';
+import { QueryResponseCache as RelayQueryResponseCache } from 'relay-runtime/lib';
```

Comment:
`import { QueryResponseCache as RelayQueryResponseCache } from 'relay-runtime';`

Reply:
In fact, I think it makes sense to change all occurrences. i.e. no need for the `as` alias.